### PR TITLE
improve support cutting planes

### PR DESCRIPTION
**Pull request by fserra**
- add functions for adding cuts
- add function to access tableau information
- add test of a separator

### CODE REVIEW

File: tests/test_gomory.py [hunk 1]
```diff
@@ -0,0 +1,339 @@
...
+    s.setIntParam("separating/closecuts/freq", -1)
+    s.setIntParam("separating/clique/freq", -1)
+    s.setIntParam("separating/zerohalf/freq", -1)
+
+    # only two rounds of cuts
+    s.setIntParam("separating/maxroundsroot", 2)
+
+    return s
+
+def test_CKS():
```
Comment (mattmilten) [inline tests/test_gomory.py]: what does `CKS` mean?

File: src/pyscipopt/scip.pyx [hunk 2]
```diff
@@ -882,6 +996,112 @@ cdef class Model:
...
+        #PY_SCIP_CALL(SCIPreleaseRow(self._scip, &row))
+        return PyRow
+
+    def getRowActivity(self, Row row):
+        return SCIPgetRowActivity(self._scip, row.row)
+
+    def getRowLPActivity(self, Row row):
+        return SCIPgetRowLPActivity(self._scip, row.row)
+
+    # TODO: do we need this? (also do we need release var??)
```
[the remark this answers is not in the document]
Comment (fserra) [reply src/pyscipopt/scip.pyx]: yeah well, apparently we need to keep it there for the moment. Unless we come up with a nice way of avoiding it, maybe by using expressions as mentioned above

File: src/pyscipopt/scip.pyx [hunk 3]
```diff
@@ -1864,7 +2088,7 @@ cdef class Model:
         presol.model = <Model>weakref.proxy(self)
         Py_INCREF(presol)
 
-    def includeSepa(self, Sepa sepa, name, desc, priority, freq, maxbounddist, usessubscip=False, delay=False):
+    def includeSepa(self, Sepa sepa, name, desc, priority = 0, freq = 10, maxbounddist = 1.0, usessubscip=False, delay=False):
```
[the remark this answers is not in the document]
Comment (fserra) [reply src/pyscipopt/scip.pyx]: done

File: tests/test_gomory.py [hunk 4]
```diff
@@ -0,0 +1,339 @@
...
+from pyscipopt import Model, Sepa, SCIP_RESULT, SCIP_PARAMSETTING
+from pyscipopt.scip import is_memory_freed
+
+class GMI(Sepa):
+
+    def __init__(self):
+        self.ncuts = 0
+
+    def getGMIFromRow(self, cols, rows, binvrow, binvarow, primsol):
+        # Given the row (binvarow, binvrow) of the tableau, computes gomory cut
```
[the remark this answers is not in the document]
Comment (fserra) [reply tests/test_gomory.py]: done

File: tests/test_gomory.py (hunk 1 above)
Comment (fserra) [reply tests/test_gomory.py]: cook kannan and schrijver

**Comment by mattmilten** [comment]
I guess, I need to disable Codecov -- or even better: correct the coverage calculation...

**Pull request merged**